modules/features/falcon_feature_gifts/falcon_feature_gifts.info.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Falcon Feature Gifts'
+description: 'Provides the Gifts'
```

Comment:
We need to update it to be more descriptive. I'd say "Charity gifts shop" or something like this.

Reply:
Fixed.